experimental/packages/sdk-logs/src/LoggerProvider.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type * as logsAPI from '@opentelemetry/api-logs';
+import { Resource } from '@opentelemetry/resources';
+import { merge } from '@opentelemetry/core';
+
+import type { LoggerProviderConfig, LogRecordLimits } from './types';
+import type { LogRecordProcessor } from './LogRecordProcessor';
+import { Logger } from './Logger';
+import { loadDefaultConfig } from './config';
+import { MultiLogRecordProcessor } from './MultiLogRecordProcessor';
+
+export class LoggerProvider implements logsAPI.LoggerProvider {
+  private readonly _loggers: Map<string, Logger> = new Map();
+  private readonly _resource: Resource;
+  private readonly _logRecordLimits: LogRecordLimits;
+  private readonly _activeProcessor: MultiLogRecordProcessor;
+
+  constructor(config: LoggerProviderConfig = {}) {
+    const { resource, logRecordLimits, forceFlushTimeoutMillis } = merge(
+      {},
+      loadDefaultConfig(),
+      config
+    );
+    this._resource = Resource.default().merge(resource ?? Resource.empty());
+    this._activeProcessor = new MultiLogRecordProcessor(
+      forceFlushTimeoutMillis
+    );
+    this._logRecordLimits = logRecordLimits;
+  }
+
+  /**
+   * Get a logger with the configuration of the LoggerProvider.
+   */
+  public getLogger(
+    name: string,
+    version?: string,
+    options?: logsAPI.LoggerOptions
+  ): Logger {
+    const { schemaUrl = '', eventDomain } = options || {};
+    const key = `${name}@${version || ''}:${schemaUrl}`;
+    if (!this._loggers.has(key)) {
+      this._loggers.set(
+        key,
+        new Logger({
+          eventDomain,
```

Comment:
This shouldn't be needed. Logs SDK shouldn't be aware of events.

Reply:
We might need to merge [this PR](https://github.com/open-telemetry/opentelemetry-js/pull/3550) before this can be merged without the events part.